automation/project/Dependencies.scala

```diff
@@ -52,7 +53,9 @@ object Dependencies {
     "org.seleniumhq.selenium" % "selenium-java" % "3.8.1" % "test",
     "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
     "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-14278f08f"
-      exclude("com.typesafe.scala-logging", "scala-logging_2.11") exclude("com.typesafe.akka", "akka-stream_2.11"),
+      exclude("com.typesafe.scala-logging", "scala-logging_2.11")
+      exclude("com.typesafe.akka", "akka-stream_2.11")
+      exclude("bio.terra", "workspace-manager-client"),
```

Comment:
workspace-manager-client was included as a transitive dependency from rawls-model. I've excluded it from rawls-model and included it explicitly above, so we have more visibility into and control over which version we are using.